homeassistant/components/heatzy/climate.py

```diff
@@ -0,0 +1,188 @@
+"""Climate sensors for Heatzy."""
+import logging
+
+from heatzypy.exception import HeatzyException
+
+from homeassistant.components.climate import ClimateEntity
+from homeassistant.components.climate.const import (
+    HVAC_MODE_HEAT,
+    HVAC_MODE_OFF,
+    PRESET_AWAY,
+    PRESET_COMFORT,
+    PRESET_ECO,
+    PRESET_NONE,
+    SUPPORT_PRESET_MODE,
+)
+from homeassistant.const import TEMP_CELSIUS
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import DOMAIN, PILOTEV1, PILOTEV2
+
+MODE_LIST = [HVAC_MODE_HEAT, HVAC_MODE_OFF]
+PRESET_LIST = [PRESET_NONE, PRESET_COMFORT, PRESET_ECO, PRESET_AWAY]
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_entry(hass, config_entry, async_add_entities):
+    """Load all Heatzy devices."""
+    coordinator = hass.data[DOMAIN][config_entry.entry_id]
+    devices = []
+    for device in coordinator.data.values():
+        product_key = device.get("product_key")
+        if product_key in PILOTEV1:
+            devices.append(HeatzyPiloteV1Thermostat(coordinator, device["did"]))
+        elif product_key in PILOTEV2:
+            devices.append(HeatzyPiloteV2Thermostat(coordinator, device["did"]))
+    async_add_entities(devices)
+
+
+class HeatzyThermostat(CoordinatorEntity, ClimateEntity):
+    """Heatzy climate."""
+
+    def __init__(self, coordinator, unique_id):
+        """Init."""
+        super().__init__(coordinator)
+        self._coordinator = coordinator
+        self._unique_id = unique_id
```

Comment:
```suggestion
        self._attr_unique_id = unique_id
        self._attr_name = coordinator.data[unique_id]["dev_alias"]
```

See: <https://developers.home-assistant.io/docs/core/entity#entity-class-attributes>